templates/upload.tmpl

```diff
@@ -0,0 +1,106 @@
+<html>
+
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
+    <link rel="stylesheet" href="https://s3.csh.rit.edu/csh-material-bootstrap/4.0.0-beta.3/dist/csh-material-bootstrap.min.css" media="screen">
+    <style>
+        html {
+            position: relative;
+            min-height: 100%;
+        }
+
+        body {
+            margin-bottom: 60px;
+            /* Margin bottom by footer height */
+        }
+
+        .footer {
+            position: absolute;
+            bottom: 0;
+            width: 100%;
+            height: 60px;
+            /* Set the fixed height of the footer here */
+            line-height: 60px;
+            /* Vertically center the text there */
+            background-color: #f5f5f5;
+        }
+    </style>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container">
+            <a class="navbar-brand" href="/upload">Plug</a>
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                    <a class="nav-link" href="/upload">Upload</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/admin">Admin <span class="sr-only">(current)</span></a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <div class="jumbotron">
+        <div class="row justify-content-center">
+            <div class="col-lg-7">
+                <div class="jumbotron" style="max-width: 800px">
+                    <h1 class="display-3">Upload a Plug!</h1>
+                    <p class="lead">You will lose 1 drink credit in exchange for a 1000 view-limit of your plug.<br> Plugs must be 728x200 pixels and in PNG, or JPG format</p>
+                    <hr class="my-4">
+
+                    <div class="form-group">
+                        <input class="form-control-file" id="exampleInputFile" aria-describedby="fileHelp" type="file">
```

Comment:
In routes.go:60 we're currently grabbing the input by it's `name` attribute (I thought it was `id`, but looking through [this](https://github.com/gin-gonic/gin/blob/master/examples/upload-file/single/public/index.html) Gin example makes me think otherwise. I'd have to check the source to be 100% certain though), we're looking for a value of `file`, which you currently have not defined. Since `file` is ambiguous maybe it would be a good idea to also edit that line so that we're looking for something a bit more clear, like `fileUpload` or `uploadedPlug`.